program/exhibited.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+order: 3
+title: "Exhibited works"
```

Comment:
How are we going to change the behaviour to link in to these? When should we link down to program vs. directly to site? When should that go live?

Reply:
I think there could be a CTA for visiting the site in both the schedule and program. Revisiting this in another PR.